applications/nrf_desktop/src/modules/Kconfig.hid

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "HID"
+
+config DESKTOP_HID_REPORT_DESC
+	string "HID report descriptor"
+	default ""
+	help
+	  This option specifies a path to the HID report descriptor file.
+
+config DESKTOP_HID_MOUSE
+	bool "Device is a HID mouse"
+	select DESKTOP_HID_REPORT_MOUSE_SUPPORT
+	help
+	  Enable reports needed by a mouse.
+
+config DESKTOP_HID_KEYBOARD
+	bool "Device is a HID keyboard"
+	select DESKTOP_HID_REPORT_KEYBOARD_SUPPORT
+	select DESKTOP_HID_REPORT_SYSTEM_CTRL_SUPPORT
+	select DESKTOP_HID_REPORT_CONSUMER_CTRL_SUPPORT
+	help
+	  Enable reports needed by a keyboard.
+
+config DESKTOP_HID_DONGLE
+	bool "Device is a HID keyboard"
+	select DESKTOP_HID_REPORT_MOUSE_SUPPORT
+	select DESKTOP_HID_REPORT_KEYBOARD_SUPPORT
+	select DESKTOP_HID_REPORT_SYSTEM_CTRL_SUPPORT
```

Comment:
I think we should select only the reports that will never be unselected here (we cannot overwrite selected values in *.conf files).

We could also set the HID report support options using default y if ... (this can be overwritten).

Reply:
As mentioned above this option is a shortcut to enable minimal set of reports for a specific device. User can always manually select report set.

Reply:
In that case should keyboard enable: `DESKTOP_HID_REPORT_SYSTEM_CTRL_SUPPORT` and `DESKTOP_HID_REPORT_CONSUMER_CTRL_SUPPORT`? (It may not be necessary for small and simple keyboard I think)